android/build.gradle

```diff
@@ -1,19 +1,37 @@
+import java.nio.file.Paths
+
+def resolveModulePath(packageName) {
```

Comment:
Isn't this addressed in [your other PR?](https://github.com/react-native-async-storage/async-storage/pull/481/files) I think this PR will land first

Reply:
Yeah, this was built on top of it. The other PR should go in first, then I'll rebase and we can get this merged.